src/pages/dashboardPage/dashboardPage.tsx

```diff
@@ -0,0 +1,50 @@
+import { type RelativePath } from '../../types/navigation'
+import { YELLOW, PINK, BLUE, GREEN, AQUA } from '../../utils/colorSchemes'
+import DashboardLayout from '../../layouts/dashboardLayout/dashboardLayout'
+import NavigationMosaic from '../../components/navigationMosaic/navigationMosaic'
+import styles from './dashboardPage.module.css'
+
+const PLACEHOLDER_HREF: RelativePath = '#'
```

Comment:
TypeScript yelled at me when I put this in the objects in the `navigationCards` array as a string literal. It said it wasn't a `RelativePath` type. I considered simply forcing it in each object:
```ts
{
  colorScheme: YELLOW,
  href: '#' as RelativePath,
  children: 'Your Games',
  key: 'games'
}
```
However, that just seemed like more code.